### PR TITLE
ZK-4664: Duplicate DOM id in Listbox with pagingPosition="both"

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

**Review by scribetw — CHANGES_REQUESTED** [review]
case

File: zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4664Test.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ */
+public class B95_ZK_4664Test extends WebDriverTestCase {
+	@Test
+	public void test() throws Exception {
+		connect();
+		waitResponse();
+		JQuery input = jq(".z-paging").eq(1).find("input");
+		sendKeys(input, Keys.BACK_SPACE);
+		waitResponse();
+		sendKeys(input, Keys.NUMPAD2);
```
Comment (plijyfes) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4664Test.java:34]: '2'

File: zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4664Test.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		assertEquals("1", activePageLabel.html().trim());
+		click(jq(".z-paging-previous"));
+		waitResponse();
+		assertEquals("0", activePageLabel.html().trim());
+		click(jq(".z-paging-next"));
+		waitResponse();
+		assertEquals("1", activePageLabel.html().trim());
+		click(jq("@button"));
+		waitResponse();
+		click(jq(".z-paging-button").eq(3));
```
Comment (plijyfes) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B95_ZK_4664Test.java:48]: comment

**Review by plijyfes — CHANGES_REQUESTED** [review]

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**